app/view_models/prior_authority/v1/event_summary.rb

```diff
@@ -53,7 +55,8 @@ def heading_keys
           'Event::Decision' => "prior_authority.events.decision_#{event.details['to']}",
           'PriorAuthority::Event::DraftSendBack' => 'prior_authority.events.draft_send_back',
           'PriorAuthority::Event::SendBack' => 'prior_authority.events.sent_back',
-          'Event::Expiry' => 'prior_authority.events.expired'
+          'Event::Expiry' => 'prior_authority.events.expired',
+          'Event::ProviderUpdated' => 'prior_authority.events.provider_updated'
```

Comment:
I guess it depends on whether Nsm Expiry/ProviderUpdated events need a different structure to PA ones, and without a crystal ball... it just sucks because this sort of refactor would break all our data in the app store. Part of me wants to bit the bullet and change `event_type` as stored in the app store to an enum that we map to classes in caseworker, rather than using the class name, as that would allow us to refactor this stuff more easily later.